assets/contact.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
I recommend putting contact.html in your root of your repository because the url to this page on Github pages will then be: https://shriji96.github.io/Basic-Portfolio/contact.html INSTEAD OF https://shriji96.github.io/Basic-Portfolio/assets/contact.html